docs/tutorial/stackup/2. smart-contract/2.1 Project-Setup.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 1
+---
+
+# Project Setup
+
+First, proceed to create a new project folder and navigate to the project root directory by running the following commands.
+
+```
+mkdir capstone_aelf && cd capstone_aelf
+```
+
+Next, proceed to run the following command to create the capstone project!
+
+```
+dotnet new aelf -n BuildersDAO
+```
+
+Proceed to open the project on your desired IDE (VSCode) and rename the existing **src/Protobuf/contract/hello_world_contract.proto** file to **BuildersDAO.proto**!
+
+Once you have renamed the file, here’s how your working directory should look like!
+
+<p align="center">
+<img src="/img/project-dir.png" alt="Portkey Login" width="400"/>
+</p>
+<p align="center">Preview of Working Directory</p>
+
+With that, we have successfully created our project!
+
+
+
+
+:::warning
+**You may SKIP the following section if you are coding in codespace instead of local environment, in the codespace template provided in this workshop already have the acs12.proto included**
```

Comment:
```suggestion
**Feel free to MOVE PAST this section if you're coding within a codespace rather than a local environment. The codespace template provided in this workshop already INCLUDES the acs12.proto file.**
```